### DISCUSSION TITLE
v2.65.0

### DISCUSSION THREAD

**Discussion by nunocoracao** [nunocoracao/blowfish#1411] [Announcements]
## What's Changed
* 📚 Remove escape characters in youtubeLite documentation by @d4rklynk in https://github.com/nunocoracao/blowfish/pull/1402
* 🔄 sync by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1403
* 🐛  shortcodes article example by @eallion in https://github.com/nunocoracao/blowfish/pull/1408
* 📌 Bump puppeteer from 22.6.2 to 22.6.3 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1405
* 👤  add users eallion.com by @eallion in https://github.com/nunocoracao/blowfish/pull/1407
* ⚙️ Bump undici from 5.28.3 to 5.28.4 by @dependabot in https://github.com/nunocoracao/blowfish/pull/1410
* 🔖 v2.65.0 by @nunocoracao in https://github.com/nunocoracao/blowfish/pull/1401

## New Contributors
* @eallion made their first contribution in https://github.com/nunocoracao/blowfish/pull/1408

**Full Changelog**: https://github.com/nunocoracao/blowfish/compare/v2.64.0...v2.65.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nunocoracao/blowfish/releases/tag/v2.65.0'>v2.65.0</a>.</em>

**Comment by eallion** [comment]
It's truly a perfect theme.
